lib/repositories/genre_repository.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+
+class GenreRepository {
+  Future<List<dynamic>> getGenreNamesById(List<num> ids) async {
```

Comment:
Should this be a list of `dynamic` type? Hmmm dangerous...

Reply:
what is life without a little risk???? you wanna know?... it's.........boooring!! jaja
done